app/views/course/assessment/submission/submissions/index.json.jbuilder

```diff
@@ -7,6 +7,7 @@ json.assessment do
   json.gamified current_course.gamified?
   json.downloadable @assessment.downloadable?
   json.passwordProtected @assessment.session_password_protected?
+  json.canViewLogs can? :view_logs, @assessment
```

Comment:
Just checking: we didn't define any abilities on `:view_logs`- are you intentionally relying on the fact that teaching staff `can? :manage, @assessment`?

I'm not too sure about the business case also: maybe can document why there is a need to hide the logs for observers?

Reply:
> Just checking: we didn't define any abilities on `:view_logs`- are you intentionally relying on the fact that teaching staff `can? :manage, @assessment`?

Yup yup.

> I'm not too sure about the business case also: maybe can document why there is a need to hide the logs for observers?

Access logs are a little more sensitive when it comes to password protected assessments, especially when there are suspected cheating cases. In those cases I think it is better only for the teaching staff to deal with, and observers have no business knowing this as well. 

---

Suggestion is that perhaps I can make explicit the  `:view_logs` permission and explain that in the abilities file - will that work?

Reply:
Will instead change it to `json.canViewLogs can? :manage, assessment` instead.